src/DataTable/tests/TableRow.test.jsx

```diff
@@ -30,49 +31,61 @@ const props = {
 };
 
 const contextValue = {
-  /* eslint-disable-next-line react/prop-types */
   renderRowSubComponent: ({ row }) => <div>{row.values.name}</div>,
   visibleColumns: [...props.cells],
 };
 
-/* eslint-disable-next-line react/prop-types */
+// eslint-disable-next-line react/prop-types
 function TableRowWrapper({ value, row }) {
   return (
     <DataTableContext.Provider value={value}>
-      <table><tbody><TableRow row={row} /></tbody></table>
+      <table>
+        <tbody>
+          <TableRow row={row} />
+        </tbody>
+      </table>
     </DataTableContext.Provider>
   );
 }
 
 describe('<TableRow />', () => {
-  const wrapper = mount(<TableRowWrapper value={contextValue} row={props} />);
   it('renders a table row', () => {
-    const row = wrapper.find('tr');
-    expect(row.length).toEqual(1);
+    const { container } = render(<TableRowWrapper value={contextValue} row={props} />);
+    const row = container.querySelector('tr');
```

Comment:
I saw you added `tr-id` for `TableRow`, then maybe it is better to find `row` by `data-testid`?

Reply:
Also consider to change other `querySelector`s to `getByDataTestId` or other suitable `testing-library` selectors